.github/workflows/ts.yml

```diff
@@ -44,3 +44,21 @@ jobs:
         run: "yarn build"
       - name: "run test"
         run: "yarn test"
+  stale-api-schema:
+    name: API schema is up to date
```

Comment:
Tricky. Perhaps `Generated API schema matches source interfaces` would convey the expectation slightly more explicitly.